lib/core/extension.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:cake_wallet/core/auth_service.dart';
+import 'package:cake_wallet/di.dart';
+import 'package:cake_wallet/routes.dart';
+import 'package:cake_wallet/src/screens/auth/auth_page.dart';
+import 'package:flutter/material.dart';
+
+const List<String> _alwaysAuthenticateRoutes = [
+  Routes.showKeys,
+  Routes.backup,
+  Routes.setupPin,
+];
+
+final _authService = getIt.get<AuthService>();
+
+extension AuthenticateRoute on BuildContext {
```

Comment:
I would say we move this as a function in `AuthService` instead of an extension on Context, as it's not that much related to context, but more to the AuthService.
and it's not good to have global variables hanging around the code, we can move `_alwaysAuthenticateRoutes` to the AuthService too